### PR TITLE
More typo corrections on merge groups for schemas

### PR DESCRIPTION
**Pull request by VisLab**

### CODE REVIEW

File: docs/source/02_Terminology.md [hunk 1]
```diff
@@ -155,14 +155,32 @@ The types of characters referred to in this specification are:
...
 | `dollar` | ASCII code 36 |
-| `plus` | ASCII code 42 |
+| `percent-sign` | ASCII code 37 |
+| `ampersand` | ASCII code 38 |
+| `single-quote ` | ASCII code 39 |
+| `left-paren` | ASCII code 40 |
+| `right-paren` | ASCII code 41 |
+| `asterick` | ASCII code 42 |
+| `plus` | ASCII code 43 |
 | `hyphen` | ASCII code 45 |
```
Comment (IanCa) [inline docs/source/02_Terminology.md:169]: comma is missing, if not intentional

Comment (VisLab) [reply docs/source/02_Terminology.md:170]: Fixed before merging.

File: docs/source/02_Terminology.md [hunk 2]
```diff
@@ -147,22 +147,41 @@ The types of characters referred to in this specification are:
...
+| `double-quote ` | ASCII code 34 |
 | `number-sign` | ASCII code 35 |
 | `dollar` | ASCII code 36 |
-| `plus` | ASCII code 42 |
+| `percent-sign` | ASCII code 37 |
+| `ampersand` | ASCII code 38 |
+| `single-quote ` | ASCII code 39 |
+| `left-paren` | ASCII code 40 |
+| `right-paren` | ASCII code 41 |
+| `asterick` | ASCII code 42 |
```
Comment (happy5214) [inline docs/source/02_Terminology.md:167]: Should be `asterisk`.

File: docs/source/02_Terminology.md [hunk 3]
```diff
@@ -147,22 +147,41 @@ The types of characters referred to in this specification are:
...
+| `semicolon` | ASCII code 59 |
+| `less-than` | ASCII code 60 |
+| `equals` | ASCII code 61 |
+| `greater-than` | ASCII code 62 |
+| `question-mark` | ASCII code 63|
+| `at-sign` | ASCII code 64 |
+| `backslash` | ASCII code 92 |
 | `caret` | ASCII code 94 |
 | `underscore` | ASCII code 95 |
+| `verbar` | ASCII code 124 |
```
Comment (happy5214) [inline docs/source/02_Terminology.md:183]: `verbar` is too abbreviated. I had to actually look up the code point to know what it was referring to. `vertical-bar` is better.

File: docs/source/07_Library_schemas.md [hunk 4]
```diff
@@ -281,28 +281,38 @@ since it is already included as a partner of `liba_1.0.0` and
...
 error is generated.
 
-Note: With the possible (and rare) exception of new `unitClasses` and `units`, partnered library schemas 
-should not have auxiliary sections except for the prologue and epilogue.
+````{admonition} Avoid new auxilliary section entries in library schemas.
+:class: warning
+**Note:** With the possible (and rare) exception of new `unitClasses` and `units`, partnered library schemas 
+should not have auxiliary sections except for the `prologue` and `epilogue`.  
+
+Auxilliary sections have information for HED tools, and new entries may require modification to
```
Comment (happy5214) [inline docs/source/07_Library_schemas.md:310]: `Auxilliary` should have one `l`.

File: docs/source/02_Terminology.md [hunk 5]
```diff
@@ -147,22 +147,41 @@ The types of characters referred to in this specification are:
...
+| `lowercase` | ASCII characters a-z |  
 | `uppercase` | ASCII characters A-Z |
 | `letters` | `lowercase` and/or `uppercase` |
 | `text` | `printable` and/or `nonascii` |
 | `digits` | 0-9 |
 | `tab` | ASCII code 09 |
 | `newline` | ASCII code 10 (linefeed) |
 | `blank` | ASCII code 32 |
+| `exclamation` | ASCII code 33 |
+| `double-quote ` | ASCII code 34 |
```
Comment (happy5214) [inline docs/source/02_Terminology.md:159]: Remove extra space between "quote" and backtick.

File: docs/source/02_Terminology.md [hunk 6]
```diff
@@ -147,22 +147,41 @@ The types of characters referred to in this specification are:
...
 | `number-sign` | ASCII code 35 |
 | `dollar` | ASCII code 36 |
-| `plus` | ASCII code 42 |
+| `percent-sign` | ASCII code 37 |
+| `ampersand` | ASCII code 38 |
+| `single-quote ` | ASCII code 39 |
```
Comment (happy5214) [inline docs/source/02_Terminology.md:164]: Remove extra space between "quote" and backtick.

File: docs/source/02_Terminology.md [hunk 7]
```diff
@@ -147,22 +147,41 @@ The types of characters referred to in this specification are:
...
+| `left-paren` | ASCII code 40 |
+| `right-paren` | ASCII code 41 |
+| `asterick` | ASCII code 42 |
+| `plus` | ASCII code 43 |
+| `comma` | ASCII code 44 |
 | `hyphen` | ASCII code 45 |
 | `period` | ASCII code 46 |
+| `slash` | ASCII code 47
```
Comment (happy5214) [inline docs/source/02_Terminology.md:172]: Add bar at end of line.

File: docs/source/02_Terminology.md [hunk 8]
```diff
@@ -147,22 +147,41 @@ The types of characters referred to in this specification are:
...
 | `colon` | ASCII code 58 |
+| `semicolon` | ASCII code 59 |
+| `less-than` | ASCII code 60 |
+| `equals` | ASCII code 61 |
+| `greater-than` | ASCII code 62 |
+| `question-mark` | ASCII code 63|
```
Comment (happy5214) [inline docs/source/02_Terminology.md:178]: Add space after "63".

**Review by happy5214 — COMMENTED** [review]
Mostly formatting, but some substantive comments too.

File: docs/source/02_Terminology.md (hunk 8 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:178]: Corrected #574

File: docs/source/02_Terminology.md (hunk 6 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:164]: Corrected #574

File: docs/source/02_Terminology.md (hunk 7 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:172]: Corrected #574

File: docs/source/02_Terminology.md (hunk 5 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:159]: Corrected #574

File: docs/source/07_Library_schemas.md (hunk 4 above)
Comment (VisLab) [reply docs/source/07_Library_schemas.md:310]: Corrected #574

File: docs/source/02_Terminology.md (hunk 3 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:183]: Corrected #574

File: docs/source/02_Terminology.md (hunk 2 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:167]: Corrected #574

File: docs/source/02_Terminology.md (hunk 1 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:170]: resolved #574

File: docs/source/02_Terminology.md (hunk 2 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:167]: resolved #574

File: docs/source/02_Terminology.md (hunk 3 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:183]: resolved #574

File: docs/source/07_Library_schemas.md (hunk 4 above)
Comment (VisLab) [reply docs/source/07_Library_schemas.md:310]: resolved #574

File: docs/source/02_Terminology.md (hunk 5 above)
Comment (VisLab) [reply docs/source/02_Terminology.md:159]: resolved #574

**Pull request merged**